libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-autocomplete/cell-editor-autocomplete.module.ts

```diff
@@ -2,11 +2,11 @@ import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { SkyAutocompleteModule } from '@skyux/lookup';
 
-import { AgGridResourcesModule } from '../../../shared/ag-grid-resources.module';
+import { SkyAgGridResourcesModule } from '../../../shared/sky-ag-grid-resources.module';
 import { SkyAgGridCellEditorAutocompleteComponent } from '../cell-editor-autocomplete/cell-editor-autocomplete.component';
 
 @NgModule({
-  imports: [FormsModule, AgGridResourcesModule, SkyAutocompleteModule],
+  imports: [FormsModule, SkyAgGridResourcesModule, SkyAutocompleteModule],
```

Comment:
The prefix `Sky-` changes are just to make every library consistent and easier to regen the library resources in one command.